packages/openactive-broker-microservice/src/sample-requests.js

```diff
@@ -0,0 +1,237 @@
+const { merge } = require('lodash');
+
+const sampleRequestConstraints = {
+  "test:testOpportunityDataShapeExpression": [
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://openactive.io/remainingUses",
+      "valueExpr": {
+        "@type": "NumericNodeConstraint",
+        "mininclusive": 2
+      }
+    },
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://openactive.io/isOpenBookingAllowed",
+      "valueExpr": {
+        "@type": "test:BooleanNodeConstraint",
+        "value": true
+      }
+    },
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://schema.org/startDate",
+      "valueExpr": {
+        "@type": "test:DateRangeNodeConstraint",
+        "minDate": "2022-05-26T12:31:31.136+00:00"
+      }
+    },
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://schema.org/eventStatus",
+      "valueExpr": {
+        "@type": "test:OptionNodeConstraint",
+        "datatype": "schema:EventStatusType",
+        "blocklist": [
+          "https://schema.org/EventCancelled",
+          "https://schema.org/EventPostponed"
+        ]
+      }
+    }
+  ],
+  "test:testOfferDataShapeExpression": [
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://openactive.io/openBookingFlowRequirement",
+      "valueExpr": {
+        "@type": "test:ArrayConstraint",
+        "datatype": "oa:OpenBookingFlowRequirement",
+        "includesAll": [
+          "https://openactive.io/OpenBookingApproval"
+        ],
+        "excludesAll": [
+          "https://openactive.io/OpenBookingAttendeeDetails",
+          "https://openactive.io/OpenBookingIntakeForm"
+        ]
+      }
+    },
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://schema.org/price",
+      "valueExpr": {
+        "@type": "NumericNodeConstraint",
+        "mininclusive": 0.01
+      }
+    },
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://openactive.io/validFromBeforeStartDate",
+      "valueExpr": {
+        "@type": "test:DateRangeNodeConstraint",
+        "maxDate": "2022-05-26T10:31:31.136+00:00",
+        "allowNull": true
+      }
+    },
+    {
+      "@type": "test:TripleConstraint",
+      "predicate": "https://openactive.io/openBookingInAdvance",
+      "valueExpr": {
+        "@type": "test:OptionNodeConstraint",
+        "datatype": "oa:RequiredStatusType",
+        "blocklist": [
+          "https://openactive.io/Unavailable"
+        ]
+      }
+    }
+  ]
+};
```

Comment:
This shouldn't be required, as `test:testOpportunityCriteria` and `test:testOpenBookingFlow` should be sufficient